tools/pipelines/templates/include-test-real-service.yml

```diff
@@ -186,7 +186,7 @@ jobs:
         inputs:
           command: 'custom'
           workingDir: ${{ parameters.testWorkspace }}
-          customCommand: 'install $(Initialize.testPackageTgz) @ff-internal/aria-logger'
+          customCommand: 'install $(Initialize.testPackageTgz) @ff-internal/aria-logger@0.0.11'
```

Comment:
@alexvy86 FYI this makes it kind of painful to update the logger - you have to come back and update this too.  Is this necessary to have as part of this change?

Reply:
Not absolutely necessary, and I see the pain point. Above I mentioned how the slower cadence of changes in the logger make me think that it shouldn't be *too* painful, and the advantage is that the pipelines are protected from immediately breaking if the changes in the logger need additional work on this side (like what happened with my change to the npm scope of the dependencies). I can also see an argument that those kinds of changes in the logger are infrequent enough that it might be better to float the version here and let the sporadic issues arise and get fixed when they do.

Just a matter of how protective we want to be with the pipelines, I suppose. But I'd be fine floating it, I'll do a new PR for that.

Reply:
My bad for not reading the PR description!  I can see it both ways too.